riscv/devices.h

```diff
@@ -71,6 +71,25 @@ class mem_t : public abstract_mem_t {
   reg_t sz;
 };
 
+class abstract_sim_if_t {
+public:
+  virtual ~abstract_sim_if_t() = default;
+  virtual bool do_load(reg_t addr, size_t len, uint8_t* bytes) = 0;
+  virtual bool do_store(reg_t addr, size_t len, const uint8_t* bytes) = 0;
```

Comment:
Does the `do_` prefix actually help us out here?  Wouldn't `load`/`store` be equally appropriate?

Reply:
Yes, but I guess that most of simulators already have `load/store` , maybe not with the same signature. I can rename it

Reply:
Noted, but I think we should go ahead and rename them.

Reply:
Renamed